pkgs/tools/package-management/rpm/default.nix

```diff
@@ -28,14 +29,17 @@ stdenv.mkDerivation rec {
     "--with-external-db"
     "--with-lua"
     "--enable-python"
+    "--enable-ndb"
+    "--enable-sqlite"
     "--localstatedir=/var"
     "--sharedstatedir=/com"
   ];
 
-  postPatch = ''
-    # For Python3, the original expression evaluates as 'python3.4' but we want 'python3.4m' here
-    substituteInPlace configure.ac --replace 'python''${PYTHON_VERSION}' ${python.executable}
+  # Small fixes for ndb on darwin
+  # https://github.com/rpm-software-management/rpm/pull/1465
+  patches = [ ./darwin.patch ];
```

Comment:
@SuperSandro2000 I think you mentioned on another PR that one could use fetchpatch. Is this the recommended solution for something like my linked PR?

It seems like for a PR there's a danger that it will change and thus change patch / hash?

Reply:
Yes, it is.

You can also directly fetch the commit https://github.com/rpm-software-management/rpm/commit/2d20e371d5e38f4171235e5c64068cad30bda557.patch

Reply:
I guess there is a danger that the upstream folks as for a change which would require me to force-push which would inadvertently kill the URL/source which would kill the build.
So it's probably better to only do that AFTER they merged it.

Reply:
> would inadvertently kill the URL/source

I don't think so. Your first commit of this PR is still available https://github.com/NixOS/nixpkgs/commit/26a04c7fe3b53648767e8d3a2b4ccc43a71c25cf.

Reply:
Woah! Well in that case... let me change that :)

Reply:
Yeah. github keeps commits like this forever.